en/airframes/airframe_reference.md

```diff
@@ -1,7 +1,7 @@
 # Airframes Reference
-> **Note** **This list is auto-generated from the source code**.
-> 
-> The **AUX** channels are only available on Pixhawk Boards (labeled with **AUX OUT**).
+> **Note**
+> **This list is auto-generated from the source code**.
```

Comment:
As it says here, the page is auto generated, so you can't modify it in the guide as it will be over-written. This specific text is in https://github.com/PX4/Firmware/blob/master/Tools/px4airframes/markdownout.py#L8

Are you saying this will help because the translators know that they shouldn't touch the Note part, so this splits it into lines?

Reply:
1. Sorry I didn't find the file when I created this PR, maybe add this file link to comment {-- --} would help?
2. No. Just Crowdin can't handle “\*\*a\*\* \*\*b\*\*”, it has no space between in crowdin, looks like "<0>a</0><0>b</0>", then "\*\*a\*\*\*\*b\*\*", after build, it will be "**ab****"

Reply:
![image](https://user-images.githubusercontent.com/4748411/78729915-efcff180-796d-11ea-9dca-deab51065311.png)

Reply:
2. Sorry, Crowdin handle it well, translator deleted that space.